From 702f23f9cdb4c46a9003e8e0756f7c0d8307f177 Mon Sep 17 00:00:00 2001
From: Chris <meischris39@hotmail.com>
Date: Sat, 17 Oct 2026 10:13:19 +0100
Subject: [PATCH] [docs] Tag the backported game info labels as v22

The backports in #29669 and #29844 kept master's @skinning_v23 tags.
Piers only defines aliases up to v22, so these six entries don't show
up in its list of v22 skinning changes.
---
 xbmc/GUIInfoManager.cpp | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/xbmc/GUIInfoManager.cpp b/xbmc/GUIInfoManager.cpp
index 01f9281..b9b290a 100644
--- a/xbmc/GUIInfoManager.cpp
+++ b/xbmc/GUIInfoManager.cpp
@@ -4545,7 +4545,7 @@ constexpr std::array<InfoMap, 88> videoplayer = {{
 ///     @return **True** if achievements are being earned in hardcore mode\,
 ///     **False** otherwise.
 ///     <p><hr>
-///     @skinning_v23 **[New Boolean Condition]** \link RetroPlayer_AchievementsHardcore `RetroPlayer.AchievementsHardcore`\endlink
+///     @skinning_v22 **[New Boolean Condition]** \link RetroPlayer_AchievementsHardcore `RetroPlayer.AchievementsHardcore`\endlink
 ///     <p>
 ///   }
 ///   \table_row3{   <b>`RetroPlayer.AchievementsProgress`</b>,
@@ -5652,7 +5652,7 @@ constexpr std::array<InfoMap, 3> container_str = {{
 ///     @return The genre of the currently selected song\, album\, movie or game in a
 ///     container.
 ///     <p><hr>
-///     @skinning_v23 **[Infolabel Updated]** \link ListItem_Genre `ListItem.Genre`\endlink extended
+///     @skinning_v22 **[Infolabel Updated]** \link ListItem_Genre `ListItem.Genre`\endlink extended
 ///     to support games
 ///     <p>
 ///   }
@@ -6636,7 +6636,7 @@ constexpr std::array<InfoMap, 3> container_str = {{
 ///                  _string_,
 ///     @return The complete Text Summary of Video or a game in a container.
 ///     <p><hr>
-///     @skinning_v23 **[Infolabel Updated]** \link ListItem_Plot `ListItem.Plot`\endlink extended
+///     @skinning_v22 **[Infolabel Updated]** \link ListItem_Plot `ListItem.Plot`\endlink extended
 ///     to support games
 ///     <p>
 ///   }
@@ -7418,7 +7418,7 @@ constexpr std::array<InfoMap, 3> container_str = {{
 ///     <p><hr>
 ///     @skinning_v18 **[New Infolabel]** \link ListItem_Platform `ListItem.Platform`\endlink
 ///
-///     @skinning_v23 **[Infolabel Updated]** \link ListItem_Platform `ListItem.Platform`\endlink
+///     @skinning_v22 **[Infolabel Updated]** \link ListItem_Platform `ListItem.Platform`\endlink
 ///     now available for games in a container.
 ///     <p>
 ///   }
@@ -7437,7 +7437,7 @@ constexpr std::array<InfoMap, 3> container_str = {{
 ///     <p><hr>
 ///     @skinning_v18 **[New Infolabel]** \link ListItem_Publisher `ListItem.Publisher`\endlink
 ///
-///     @skinning_v23 **[Infolabel Updated]** \link ListItem_Publisher `ListItem.Publisher`\endlink
+///     @skinning_v22 **[Infolabel Updated]** \link ListItem_Publisher `ListItem.Publisher`\endlink
 ///     now available for games in a container.
 ///     <p>
 ///   }
@@ -7448,7 +7448,7 @@ constexpr std::array<InfoMap, 3> container_str = {{
 ///     <p><hr>
 ///     @skinning_v18 **[New Infolabel]** \link ListItem_Developer `ListItem.Developer`\endlink
 ///
-///     @skinning_v23 **[Infolabel Updated]** \link ListItem_Developer `ListItem.Developer`\endlink
+///     @skinning_v22 **[Infolabel Updated]** \link ListItem_Developer `ListItem.Developer`\endlink
 ///     now available for games in a container.
 ///     <p>
 ///   }
-- 
2.53.0

